core/emitter.js

```diff
@@ -21,6 +19,7 @@ class Emitter extends EventEmitter {
   constructor() {
     super();
     this.listeners = {};
+    EMITTERS.push(this);
```

Comment:
@jhchen we could possible iterate through `EMITTERS` here, looking for any with a `container` which is no longer in the DOM tree. this isn't straight forward, though, because `document.body.contains(node)` is false for nodes in shadow trees. we would have to go up the tree (`getRootNode`) until we reach a node which is in `document` , if ever.

Reply:
and if you don't have `getRootNode`, we would just... use `parentNode` i suppose.